CMake/GNUCompilers.cmake

```diff
@@ -41,8 +41,8 @@ if(CMAKE_CXX_FLAGS MATCHES "-march=" OR CMAKE_C_FLAGS MATCHES "-march=")
   endif() #(CMAKE_CXX_FLAGS MATCHES "-march=" AND CMAKE_C_FLAGS MATCHES "-march=")
 else() #(CMAKE_CXX_FLAGS MATCHES "-march=" OR CMAKE_C_FLAGS MATCHES "-march=")
   # use -march=native
-  SET(CMAKE_C_FLAGS     "${CMAKE_C_FLAGS} -march=native")
-  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=native")
+  #SET(CMAKE_C_FLAGS     "${CMAKE_C_FLAGS} -march=native")
```

Comment:
This change slipped.

Reply:
If you have the problem with native on power, please submit a separate PR transferring the https://github.com/QMCPACK/miniqmc/pull/87

Reply:
Yup, just discovered that. Backing it out.